src/Dapr.Extensions.Configuration/DaprConfigurationStoreExtension.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using Dapr.Client;
+using Microsoft.Extensions.Configuration;
+
+namespace Dapr.Extensions.Configuration
+{
+    /// <summary>
+    /// Extension used to call the Dapr Configuration API and store the values in a <see cref="IConfiguration"/>.
+    /// </summary>
+    [Obsolete]
+    public static class DaprConfigurationStoreExtension
```

Comment:
Do we have an existing class with extension methods for `IConfigurationBuilder`? I expected to see these methods added to that class.

Reply:
We don't have any classes that extend `IConfigurationBuilder` as far as I can tell. I followed what the `DaprSecretStoreConfigurationExtensions.cs` class did.